src/js/components/Drop/DropContainer.js

```diff
@@ -314,6 +315,7 @@ const DropContainer = forwardRef(
       <StyledDrop
         ref={ref || dropRef}
         as={Box}
+        background={background}
```

Comment:
On first glance, this seems like it should work similar to `elevation` below. Or, perhaps StyledDrop should take care of checking for `plain` for `elevation`?

Reply:
`background` is a little tricky since it is being used twice, once in DropContainer.js (line 338), just to determine whether the background is dark or not (regardless of `plain` property), and the second time is on StyledDrop where the actual style is being applied (line 44).
I tried not to reinvent the wheel and maintain backward compatibility of the theme 'background' style that already existed, by keeping the existing behavior and only enhancing it to support the `background` prop. So the `plain` logic on the prop is kept intact.